anime_downloader/sites/4anime.py

```diff
@@ -0,0 +1,44 @@
+import logging
+from anime_downloader.sites.anime import Anime, AnimeEpisode, SearchResult
+from anime_downloader.sites import helpers
+
+logger = logging.getLogger(__name__)
+
+class Anime4(Anime, sitename = '4anime'):
+    sitename = '4anime'
+
+    @classmethod
+    def search(cls, query):
+        data = {
+            "action": "ajaxsearchlite_search", 
+            "aslp": query, 
+            "asid": 1, 
+            "options": "qtranslate_lang=0&set_intitle=None&customset%5B%5D=anime"
+            }
+        soup = helpers.soupify(helpers.post("https://4anime.to/wp-admin/admin-ajax.php", data=data).text)

```

Comment:
This can be simplified to 
```
        search_results = helpers.soupify(helpers.post("https://4anime.to/wp-admin/admin-ajax.php", data=data)).select('div.info > a')

        search_results = [
            SearchResult(
                title = i.text,
                url = i['href']
                )
            for i in search_results
            ]
        return search_results
```

Reply:
nice